### PR TITLE
Feature/arc 1500 request list tests

### PR DESCRIPTION
**Pull request by EmileVanTichelenSHD**
https://meemoo.atlassian.net/browse/ARC-1500

I also changed some things from send request list to make it cleaner / more optimized.

### CODE REVIEW

File: src/modules/campaign-monitor/services/campaign-monitor.service.spec.ts [hunk 1]
```diff
@@ -69,8 +81,10 @@ const getMockVisit = (): Visit => ({
 
 describe('CampaignMonitorService', () => {
 	let campaignMonitorService: CampaignMonitorService;
+	const env = process.env;
```
Comment (tourlbr) [inline src/modules/campaign-monitor/services/campaign-monitor.service.spec.ts:84]: REQ: this should no longer be required so please remove this.

File: src/modules/campaign-monitor/services/campaign-monitor.service.spec.ts [hunk 2]
```diff
@@ -69,8 +81,10 @@ const getMockVisit = (): Visit => ({
...
 
 	beforeEach(async () => {
+		process.env = { ...env };
```
Comment (tourlbr) [inline src/modules/campaign-monitor/services/campaign-monitor.service.spec.ts:87]: REQ: this should no longer be required so please remove this.

**Review by tourlbr — CHANGES_REQUESTED** [review]

**Review by tourlbr — APPROVED** [review]

File: src/modules/campaign-monitor/templates/templates.service.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Template } from '../campaign-monitor.types';
+
+export class TemplateService {
```
Comment (tourlbr) [inline src/modules/campaign-monitor/templates/templates.service.ts:3]: REQ: You can just do similar logic as you dit with xml & csv. No need for Class.

**Review by tourlbr — CHANGES_REQUESTED** [review]

**Review by tourlbr — APPROVED** [review]

**Review by bertyhell — APPROVED** [review]

File: src/modules/campaign-monitor/services/campaign-monitor.service.ts [hunk 4]
```diff
@@ -203,15 +204,15 @@ export class CampaignMonitorService {
 
 	public async sendTransactionalMail(
 		emailInfo: CampaignMonitorSendMailDto
-	): Promise<void | BadRequestException> {
+	): Promise<void | BadRequestException | boolean> {
```
Comment (bertyhell) [inline src/modules/campaign-monitor/services/campaign-monitor.service.ts:207]: We shouldn't add exceptions in the return type
```suggestion
	): Promise<boolean> {
```

**Review by bertyhell — CHANGES_REQUESTED** [review]

**Review by bertyhell — CHANGES_REQUESTED** [review]

**Pull request merged**